src/ApiService/FunctionalTests/1f-api/Helpers.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace FunctionalTests {
+    public class Helpers {
+
+        public static async Task<(Pool, Scaleset)> CreatePoolAndScaleset(PoolApi poolApi, ScalesetApi scalesetApi, string os = "linux", string? region = null, int numNodes = 2) {
+            var image = (os == "linux") ? ScalesetApi.Image_Ubuntu_20_04 : ScalesetApi.ImageWindows;
+
+            var newPoolId = Guid.NewGuid().ToString();
+            var newPoolName = PoolApi.TestPoolPrefix + newPoolId;
+            var newPool = await poolApi.Create(newPoolName, os);
+
+            Assert.True(newPool.IsOk, $"failed to create new pool: {newPool.ErrorV}");
```

Comment:
In a recent PR, I added the `FluentAssertions` library so we can do things like

`newPool.IsOk.Should().BeTrue()`

I'm not saying you should change it here, just spreading the message 😛

Reply:
I'll start doing this. There is more tests to write, so I'll redo these ones as well